p2p/addrbook.go

```diff
@@ -169,6 +169,18 @@ func (a *AddrBook) OurAddresses() []*NetAddress {
 	return addrs
 }
 
+// List returns a list of new and old addresses.
+func (a *AddrBook) List() []*knownAddress {
```

Comment:
Public function with private return args. I think func should be like `GetSelection`

Reply:
Since I need the additional information provided by knownAddress, would you like the function to be private instead?

Reply:
no. let's make `knownAddress` public. also returning knownAddresses and not NetAddress creates a dissonance (maybe there is a better word for it). i.e. imagine some array ds with API:

```
all() -> []innerElem
some() -> []Element
```

let's rename `List` to `ListOfKnownAddresses()`

Reply:
Okay.